.github/workflows/packer-build-and-publish.yml

```diff
@@ -233,11 +317,126 @@ jobs:
           fi
 
       - name: Publish release
+        id: publish
         uses: softprops/action-gh-release@c062e08bd532815e2082a85e87e3ef29c3e6d191 # v2.0.8
         with:
-          files: artifacts/**/*.json
+          files: |
+            artifacts/**/*.json
+            artifacts/**/*.md
           body: |
-            Release ${{ github.ref_name }} from workflow [#${{ github.run_id}}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})
-          draft: ${{ needs.check-if-main.outputs.is-main != 'true' || contains(github.ref_name, '-') }}
-          make_latest: ${{ !(github.base_ref != 'main' || contains(github.ref_name, '-')) && steps.is_latest.outputs.value }}
+            Release ${{ github.ref_name }} from workflow [#${{ github.run_id }}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})
+          draft: ${{ needs.check-main.outputs.is-main != 'true' || contains(github.ref_name, '-') }}
+          make_latest: ${{ !(needs.check-main.outputs.is-main != 'true' || contains(github.ref_name, '-')) && steps.is_latest.outputs.value }}
           generate_release_notes: true
+
+  share-and-copy-amis:
+    needs:
+      - check-main
+      - publish
+    runs-on: ubuntu-latest
+
+    env:
+      DEV_ACCOUNT: "654654387067"
+      PROD_ACCOUNT: "654654387067"
```

Comment:
these are the same

```suggestion
      DEV_ACCOUNT: "654654387067"
      PROD_ACCOUNT: "590183878691"
```

Reply:
This is just for testing purposes, it will be reverted once I revert https://github.com/grafana/runner-images/pull/125/commits/84460294aca461ea11e97e7e74f746b11d3d56c1

Reply:
Added todo comment to make that clearer.